lib/hooks/digest.cpp

```diff
@@ -0,0 +1,224 @@
+/** Digest hook.
+ *
+ * @author Philipp Jungkamp <philipp.jungkamp@opal-rt.com>
+ * @copyright 2023, OPAL-RT Germany GmbH
+ * @license Apache 2.0
+ *********************************************************************************/
```

Comment:
Part of a cleanup which I am currently performing in another PR
```suggestion
/* Digest hook.
 *
 * Author: Philipp Jungkamp <philipp.jungkamp@opal-rt.com>
 * SPDX-FileCopyrightText: 2023 OPAL-RT Germany GmbH
 * SPDX-License-Identifier: Apache-2.0
 */
```